packages/inspire/InspireGateway.js

```diff
@@ -129,13 +130,23 @@ export default class InspireGateway extends LogEventGenerator {
         extendValaaSpaceWithInspire(rootScope, hostDescriptors, defaultAuthorityConfig, engine);
       }
       rootScope.Valaa.gateway = this;
-      ret[viewName] = new InspireView({ engine, name: `${viewConfig.name} View` })
-          .initialize(viewConfig);
-      this.warnEvent(`Opened InspireView ${viewName}`,
-          ...(!this.getDebugLevel() ? [] : [", with:",
-            "\n\tviewConfig:", viewConfig,
-            "\n\tview:", ret[viewName],
-          ]));
+      if (isInBrowser) {
```

Comment:
Let's take this one step further.

Add a `, createView = function (options) => new InspireView(options)`
as a parameter and have the `ret[viewName] = createview(...)` use it.
This retains full compatibility with inspire behavior in browser contexts.
Move the PerspireView import all the way back to @valos/inspire/index.js export:
`export { PerspireView } from "..."`
Finally,  in valma-perspire, do const PerspireView = require("@valos/inspire").PerspireView; and give an appropriate createView as parameter to the createAndConnectViewsToDOM.

What this achieves is that when deploying inspire to browser contexts, webpack can prune all perspire-related code and we avoid bloating the inspire gateway runtime. I talked about plugins, but this is even simpler way to get this.